### ISSUE TITLE
[UX] The "Add user account" menu link should be under "People"

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#550]
I see no reason for it to be at the top level. We do not list "Add content" or "Install new module/theme" there, so why are we special-casing that?

**Comment by klonos** [comment]
...also, can we please change the view label and the menu bar link from "Administer users" to "Manage users"?

**Comment by klonos** [comment]
I honestly don't know what's wrong with my setup. It is an old local installation that I use in order to test commits as they happen. So it was initially some version from git and eventually updated to 1.0.0-preview and then to latest git. Each time after updating, I always made sure to run update.php when needed and to flash caches too.

I just tested with a vanilla 1.0.0-preview and then upgraded to latest git and I cannot reproduce the issue there. I compared the ```schema_version``` column from this installation to the one with the issue and all modules seem to be in the same latest version...

- color, field, filter, search, contextual, text and views_ui are all ```1000```
- update module is ```1001```
- menu, block and comment modules are ```1002```
- views module is ```1003```
- taxonomy module is ```1004```
- image module is ```1005```
- node module is ```1008```
- user module is ```1009```
- system module is ```1033```

In other words, I don't think I missed running any db updates.

So, I went ahead and copied my settings.php from the working installation over to the one with the issue. Upon page refresh I got this:

```
Fatal error: Call to undefined function menu_get_custom_theme() in C:\www\backdrop\core\includes\theme.inc on line 101
Call Stack
#  Time    Memory   Function                        Location
1  0.0092  1221784  _backdrop_exception_handler( )  ..\bootstrap.inc:0
2  0.0101  1280840  _backdrop_log_error( )          ..\bootstrap.inc:2554
3  0.0462  4831096  theme( )                        ..\errors.inc:242
4  0.0462  4831264  theme_get_registry( )           ..\theme.inc:1003
5  0.0462  4831952  backdrop_theme_initialize( )    ..\theme.inc:280
```

Don't know if that helps at all or if it is due to the "unholy" thing I've done.

**Comment by quicksketch** [comment]
> Fatal error: Call to undefined function menu_get_custom_theme() in C:\www\backdrop\core\includes\theme.inc on line 101

This happens when `settings.php` is pointing at an empty database and config files still exist. Be sure to actually empty both the database and the config directory when trying a fresh install. We've got an issue for this at #531 .

**Comment by klonos** [comment]
Well, the old config files are indeed there (intentionally), but the db that settings.php points to is not empty. What I actually tried to do is to point the faulty installation to the db of the working one while still keeping the old config files (both installations were pretty much vanilla). In short, I tried swapping the db.

Anyways, I think that this is similar to (but not the same as) #470. The error you get at least is the same.
